sdk/extensions/Azure.Extensions.AspNetCore.DataProtection.Blobs/src/AzureBlobXmlRepository.cs

```diff
@@ -26,7 +26,7 @@ namespace Azure.AspNetCore.DataProtection.Blobs
     /// <remarks>
     /// Instances of this type are thread-safe.
     /// </remarks>
-    public sealed class AzureBlobXmlRepository : IXmlRepository
+    internal sealed class AzureBlobXmlRepository : IXmlRepository
```

Comment:
Are these GA'ed?

Reply:
I wrote this half a second after I commented on updating the version to `-preview.2`.  I'm clearly not of sound mind...